src/main/java/Duke/Tasks.java

```diff
@@ -0,0 +1,29 @@
+package Duke;
+
+public class Tasks {
+    String taskName;
+    boolean done;
+    public Tasks() {
+        taskName = "";
+        done = false;
+    }
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public String taskDoneStatus() {

```

Comment:
Names representing methods should be verbs, as per the Java coding standard. getTaskDoneStatus() would be better.